src/components/Loader/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from "./Loader";
```

Comment:
```suggestion
export { default, LoaderSize } from "./Loader";
```

тогда тебе не придётся при импорте LoaderSize писать import { LoaderSize } from 'components/Loader/Loader', можно будет ограничиться путём 'components/Loader'